assets/admin/editor-wizard/patterns-list.scss

```diff
@@ -17,6 +17,7 @@ $thumbnail-height: 235px;
 
 		&-preview {
 			border: 1px solid #f0f0f0;
+			background: var(--sensei-background-color-global, var(--sensei-course-theme-background-color, var(--wp--preset--color--background, #FFFFFF)));
```

Comment:
I tested activating the theme Twenty Twenty-Four, and changing the background, but the `--wp--preset--color--background` didn't work there. I noticed it was using the variable `--wp--preset--color--base` instead. So searching in the documentation, I found a warning [here](https://developer.wordpress.org/themes/global-settings-and-styles/settings/color/#custom-color-palette) saying that it's recommended to use the `base` for the site background.

Was there a theme you tested that was using the `--wp--preset--color--background` as site background? If so, maybe we should use both in the callback chain?